tests/test_threadlocal.py

```diff
@@ -0,0 +1,87 @@
+# All Rights Reserved.
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+"""Tests for threadlocal dict module."""
+
+import random
+import string
+import threading
+import Queue
+import unittest
+
+from simpl import threadlocal
+
+
+class TestThreadLocal(unittest.TestCase):
+
+    def setUp(self):
+        # clear the threadlocal dict
+        threadlocal.get_context().clear()
+
+    def get_some_text(self, length=16):
+
```

Comment:
Methods & functions should have docstrings (PEP-8) and there should be no blank lines before or after docstrings for methods/functions (PEP-257). So even if we omit the docstring in a test module, I think the blank lines at that beginning of the method should be removed.